### PR TITLE
Implement getStats()

### PR DESCRIPTION
**Pull request by spkrka**

### CODE REVIEW

File: folsom/src/main/java/com/spotify/folsom/MemcachedStats.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.spotify.folsom;
```
Comment (protocol7) [inline folsom/src/main/java/com/spotify/folsom/MemcachedStats.java:16]: Missing copyright header

**Comment by protocol7** [comment]
Needs a few more tests for the request classes and parsers.

File: folsom/src/main/java/com/spotify/folsom/MemcachedStats.java (hunk 1 above)
Comment (spkrka) [reply folsom/src/main/java/com/spotify/folsom/MemcachedStats.java:16]: Will fix

**Comment by spkrka** [comment]
Yes, some more unit tests might make sense, as opposed to only having integration tests

I will also try to prepare semantic metrics support (as a separate PR) before merging this, to be more certain this approach makes sense.

**Pull request merged**